src/frontend/src/tables/general/ExtraLineItemTable.tsx

```diff
@@ -27,11 +27,13 @@ import {
 export default function ExtraLineItemTable({
   endpoint,
   orderId,
+  orderDetailRefresh,
   currency,
   role
 }: Readonly<{
   endpoint: ApiEndpoints;
   orderId: number;
+  orderDetailRefresh: any;
```

Comment:
One small thing here regarding typing:

Instead of:

```typescript
orderDetailRefresh: any;
```

it should be

```typescript
orderDetailRefresh: () => void
```

(please roll this out to all tables where you have added this parameter

Reply:
I have added this